### ISSUE TITLE
Wrong absolute path in configs - SASS styles

### ISSUE THREAD

**Issue by vbatrla** [coryhouse/react-slingshot#594]
I have found wrong path in file **webpack.config.dev.js** and **webpack.config.prod.js** in plugin sass-loader. Take a look at line 116 of this file. The problem should be in includePath parameter. I'm guessing there should be option "styles" in third parameter instead of "scss". If I'm wrong just let me know why. Thank you.

```
{
  loader: 'sass-loader',
  options: {
	includePaths: [path.resolve(__dirname, 'src', 'scss')],
	sourceMap: true
}      
```

**Comment by coryhouse** [comment]
Ah, good point. I think it makes sense to omit the last arg and just point it at src. Would you be willing to do a PR?

**Comment by kwelch** [comment]
Closed in #615
